### PR TITLE
Uncommented apps that are useful for every user.

### PR DESCRIPTION
**Pull request by BoQsc**
- Windows Sound recorder 
- Windows Camera
- Windows Alarms
are useful Apps for most of the script users.

### CODE REVIEW

File: scripts/remove-default-apps.ps1 [hunk 1]
```diff
@@ -27,12 +27,12 @@ $apps = @(
     "Microsoft.People"
     "Microsoft.SkypeApp"
     #"Microsoft.Windows.Photos"
-    "Microsoft.WindowsAlarms"
+    #"Microsoft.WindowsAlarms"
```
Comment (BoQsc) [inline scripts/remove-default-apps.ps1]: Windows Alarms is not a bloatware. There is not many efficient apps in the Windows store to replace such useful thing.

File: scripts/remove-default-apps.ps1 [hunk 2]
```diff
@@ -27,12 +27,12 @@ $apps = @(
...
     #"Microsoft.WindowsCalculator"
-    "Microsoft.WindowsCamera"
+    #"Microsoft.WindowsCamera"
```
Comment (BoQsc) [inline scripts/remove-default-apps.ps1]: Useful for testing laptop's cameras. Efficient program for taking screenshots. It's core application not a bloatware. 
By removing this APP, most of the laptop computers lose ability to take photos and record video.
By removing this, computer loses component functionality as, there is not many apps to replace this one.

File: scripts/remove-default-apps.ps1 [hunk 3]
```diff
@@ -27,12 +27,12 @@ $apps = @(
...
     "Microsoft.WindowsMaps"
     "Microsoft.WindowsPhone"
-    "Microsoft.WindowsSoundRecorder"
+    #"Microsoft.WindowsSoundRecorder"
```
Comment (BoQsc) [inline scripts/remove-default-apps.ps1]: A good APP for testing microphone for the laptop and quickly recording voice/sound.

**Comment by majkinetor** [comment]
This is highly opinionated. Clearly the list of scripts that are executed without any params is not going to fit all. I personally do not use store apps at all, and the more removed the better.

**Comment by W4RH4WK** [comment]
I am with @majkinetor on this one, I do not like the whole App thing at all, and I am even thinking about removing Photos and Calculator by default, to add some *native* application instead.

Side note: Colleague of mine at work has mind boggling issues with Apps regarding DPI scaling on his laptop as well as remembering the correct keyboard layout on an up-to-date Windows (no creators update).

**Comment by W4RH4WK** [comment]
I'll think about it in more detail. Anyway, thank you for your contribution.

**Comment by majkinetor** [comment]
@BoQsc when time permits me, I intended to create more customizable fork of this project - i.e. take all the things that exist here and tweak a few things from config file or script, like this one you want.

I think this is the best option. I see this project as 'lets-collect-them-all-and-test-on-massive-scale' type that can lead to creation of more user friendly version.

**Comment by W4RH4WK** [comment]
@majkinetor (offtopic) drop me a note when your project goes live, I'd like to point it out in the README of this repository.

**Pull request closed without merging**